totalRP3/modules/chatframe/chatframe.lua

```diff
@@ -384,70 +396,34 @@ local function detectEmoteAndOOC(message, NPCEmoteChatColor, isEmote)
 		NPCEmoteChatString = NPCEmoteChatColor:GetColorCodeStartSequence();
 	end
```

Comment:
Unrelated but might be worth cleaning up one day; with the `|r` change in 9.0.1 to pop colours individually rather than reset them I think this code as-is would lead to a situation where, for NPC talk, we'd be pushing the emote colour onto the colour stack anew each time we finish an OOC/Speech substitution and if there were enough of them we'd hit the limit and break further colours in the message.